app/controllers/merchant/items_controller.rb

```diff
@@ -1,4 +1,25 @@
 class Merchant::ItemsController < Merchant::BaseController
   def index
+    @merchant = Merchant.find(current_user.merchant_id)
+    @items = @merchant.items
+    @top_five_items = Item.items_by_popularity(5)
+    @bottom_five_items = Item.items_by_popularity(5, 'asc')
+    render "items/index"
```

Comment:
I was able to redirect to the existing items view here as opposed to creating a merchant/items/index. There were no authorization issues.

Reply:
Nice. So you just specified which items were @items which allowed you to bypass creating a new view? and when you redirect is the path ```/merchant/items```? Just to clarify I'm assuming this is for a merchant-specific item index